rust/ql/test/utils-tests/modelgenerator/CaptureSummaryModels.ql

```diff
@@ -0,0 +1,11 @@
+import rust
+import utils.modelgenerator.internal.CaptureModels
+import utils.test.InlineMadTest
+
+module InlineMadTestConfig implements InlineMadTestConfigSig {
+  string getCapturedModel(Function c) { result = captureMixedFlow(c, _) }
```

Comment:
Again, I think we just want `ContentSensitive::captureFlow` for Rust.

Reply:
Makes no difference in the tests, so let's just do that yes 👍